[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points

b-koopman opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #65241

Differential Revision: [D31150087](https://our.internmc.facebook.com/intern/diff/D31150087)

b-koopman committed:
[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points

[ghstack-poisoned]

jingsh reviewed (commented):
generally looks good, except comments on the tests being commented out

b-koopman commented on benchmarks/operator_benchmark/pt/quantization_test.py:
```diff
@@ -59,11 +59,11 @@ def init(self, C, M, N, dtype, mode):
 
-
+'''
 op_bench.generate_pt_test(
     quantize_per_tensor_configs_short + quantize_per_tensor_configs_long,
     QuantizePerTensorBenchmark)
-
+'''
```
Thanks, will un-comment.

EdwaRen reviewed (approved):
lgtm!

b-koopman committed:
Update on "[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points"




[ghstack-poisoned]

jingsh reviewed (commented):
I think there're a few more tests got commented out, uncomment those?

line 118, 167

b-koopman committed:
Update on "[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points"




[ghstack-poisoned]

jingsh reviewed (approved):
lgtm!

b-koopman commented:
@b-koopman has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31150087).

b-koopman committed:
Update on "[quant] Add op benchmark for CPU FakeQuantizePerChannel with float zero_points"


Differential Revision: [D31150087](https://our.internmc.facebook.com/intern/diff/D31150087)

[ghstack-poisoned]

b-koopman commented:
@b-koopman has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31150087).
